modules/reference/pages/rpk/rpk-redpanda/rpk-redpanda-start.adoc

```diff
@@ -95,7 +95,7 @@ numbers, each with optional fraction and a unit suffix, such as
 
 |--tune |- |When present will enable tuning before starting Redpanda.
 
-|--well-known-io |string |The cloud provider and VM type, in the format <provider>:<vm type>:<storage type>.
+|--well-known-io |string |The cloud vendor and VM type, in the format <vendor>:<vm type>:<storage type>.
```

Comment:
```suggestion
|--well-known-io |string |The cloud vendor and VM type, in the format `<vendor>:<vm type>:<storage type>`.
```
From a quick glance of our docs - we don't typically use a hyphen inside placeholders in the rpk reference... do we?

Reply:
Yeah that page needs a bit more love. I'll add some styling to it to better match our docs, even though they don't exist in the automation.